blivet/devices/storage.py

```diff
@@ -300,7 +300,7 @@ def updateSysfsPath(self):
         # ValueErrors and EnvironmentErrors that may be raised. So catch
         # those errors, but also catch DeviceNotFoundError, which is the
         # appropriate exception and which it may raise in future.
-        except (pyudev.DeviceNotFoundError, EnvironmentError, ValueError) as e:
+        except (pyudev.DeviceNotFoundError, EnvironmentError, ValueError, OSError) as e:
```

Comment:
I think we should either add ``OSError`` to the comment above the ``except`` line make the comment more generic instead of listing all exceptions that may be raised and duplicating the list there.